pkg/core/interop_system.go

```diff
@@ -269,10 +270,13 @@ func runtimeNotify(ic *interop.Context) error {
 	// outside of the interop subsystem anyway. I'd probably fail transactions
 	// that emit such broken notifications, but that might break compatibility
 	// with testnet/mainnet, so we're replacing these with error messages.
-	_, err := stackitem.SerializeItem(elem.Item())
+	_, size, err := stackitem.SerializeItem(elem.Item())
```

Comment:
Rather you need some `buf, err := ...; if len(buf) > MaxNotificationSize` here.

Reply:
We can do `buf, err := ...; if len(buf) > MaxNotificationSize`, but then the behaviour might be different from C# in some cases.